sentry/src/main/java/io/sentry/SentryBaseEvent.java

```diff
@@ -444,6 +445,15 @@ public boolean deserializeValue(
           return true;
         case JsonKeys.EXTRA:
           Map<String, Object> deserializedExtra = (Map<String, Object>) reader.nextObjectOrNull();
+          if (deserializedExtra != null) {
```

Comment:
Should we do this in `CollectionUtils.newConcurrentHashMap` to apply to all places? I presume this problem could happen anywhere we create a `ConcurrentHashMap`.